### PR TITLE
Update incidentfield-Darktrace_Model_Breach_CommentCount.json

### PR DESCRIPTION
**Pull request by edik24**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
We are missing RN. Plus I am not sure this field is needed, I don't see it used anywhere...

**Comment by edik24** [comment]
updated version and release notes.
I agree - not sure the field is needed. but didn't want to delete it, as it is partner pack. 
maybe it is used in some customer playbook?

File: Packs/Darktrace/ReleaseNotes/2_0_6.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Darktrace Model Breach Comment Count
+- Fixed display name of incident field incident_darktracemodelbreachcommentcount
```
Comment (ShahafBenYakir) [inline Packs/Darktrace/ReleaseNotes/2_0_6.md:5]: This is not right, work with the SDK command to generate the right message

Comment (edik24) [reply Packs/Darktrace/ReleaseNotes/2_0_6.md]: fixed it now

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**